index.js

```diff
@@ -17,8 +21,14 @@ class MergeIntoFile {
 
   apply(compiler) {
     if (compiler.hooks) {
-      const plugin = { name: 'MergeIntoFile' };
-      compiler.hooks.emit.tapAsync(plugin, this.run.bind(this));
+      if (webpackMajorVersion < 5) {
+        compiler.hooks.emit.tapAsync(plugin, this.run.bind(this));
+      } else {      
+        compiler.hooks.compilation.tap(plugin, this.run.bind(this));
```

Comment:
This should be `thisCompilation` instead of `compilation` I guess, see https://github.com/FormidableLabs/webpack-stats-plugin/issues/52#issuecomment-712737439

Reply:
@markshapiro @kelunik 
See https://github.com/markshapiro/webpack-merge-and-include-globally/commit/9742cac6717ab53d2bd14f0cd57fc1b49d71d4f9

Reply:
@kelunik changed & published, thanks

Reply:
Thanks!